src/components/Dashboard/MaskPercTile.tsx

```diff
@@ -8,18 +8,49 @@ import {
   LegendContainer,
   PercentageContainer,
 } from './MaskElements';
+import useInterval from '@use-it/interval';
+import store from '../../store';
 import { PieChart, Pie } from 'recharts';
 
-const data = [
-  { name: 'Group A', value: 318, fill: '#004d00' },
-  { name: 'Group B', value: 34, fill: '#990000' },
-];
-
 const theme = {
   main: '#004d00',
 };
 
+const MaskCount = () => {
```

Comment:
since we're using typescript lets try to type as many things as possible, this line can become

`const MaskCount: React.FC = (): ReactElement => {`

Reply:
By the way `React.FC` is just an alias for `React.FunctionComponent`. They're both the same thing.

Reply:
need to look at this after final